<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SalesDialog</class>
 <widget class="QDialog" name="SalesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>718</width>
    <height>619</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sales</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog, QTableWidget{
background-color:rgb(54,54,54);
color:white;
font:12px;
}
QFrame#toolsFrame{
border:1px solid grey;
font:12px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2"/>
   </item>
   <item>
    <widget class="QFrame" name="toolsFrame">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>60</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
border-radius:20px;
color:white;
	font: 12pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(0, 227, 110);
}
QPushButton:hover{
border:2px solid white;
}
QPushButton:pressed{
border:3px solid grey;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>9</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">font:14px;
color:white;</string>
        </property>
        <property name="text">
         <string>Today's Sales</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="styleSheet">
         <string notr="true">font:14px;
color:white;</string>
        </property>
        <property name="text">
         <string>Filter:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font:14px;
background-color:transparent;
border:1px solid white;
color:white;</string>
        </property>
        <item>
         <property name="text">
          <string>All Payment Types</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Cash Payments</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>M~Pesa Payments</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>M~Pesa + Cash Payments</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Credit Payments</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Ceque Payments</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="salesonCB">
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font:14px;
background-color:transparent;
border:1px solid white;
color:white;</string>
        </property>
        <item>
         <property name="text">
          <string>Today</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Choose Date</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Choose Period</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="reloadBtn">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>  Reload  </string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>550</width>
       <height>200</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_13">
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_12">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QWidget" name="cashWdg" native="true">
              <property name="minimumSize">
               <size>
                <width>260</width>
                <height>40</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">*{
color: rgb(255, 255, 255);
font: 12pt &quot;MS Shell Dlg 2&quot;;
}
QWidget{
background-color: rgb(66, 133, 200);
}</string>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QLabel" name="label_3">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Cash Sales : </string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="cashLbl">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>0.0</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="mpesaWdg" native="true">
              <property name="minimumSize">
               <size>
                <width>260</width>
                <height>40</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">*{
color: rgb(255, 255, 255);
font: 12pt &quot;MS Shell Dlg 2&quot;;
}
QWidget{
background-color: rgb(66, 133, 200);
}</string>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <item>
                <widget class="QLabel" name="label_5">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Mpesa Sales : </string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="mpesaLbl">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>0.0</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="chequeWdg" native="true">
              <property name="minimumSize">
               <size>
                <width>260</width>
                <height>40</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">*{
color: rgb(255, 255, 255);
font: 12pt &quot;MS Shell Dlg 2&quot;;
}
QWidget{
background-color: rgb(66, 133, 200);
}</string>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_7">
               <item>
                <widget class="QLabel" name="label_11">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Cheque Sales : </string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="chequeLbl">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>0.0</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QWidget" name="cashmpesaWdg" native="true">
              <property name="minimumSize">
               <size>
                <width>260</width>
                <height>40</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">*{
color: rgb(255, 255, 255);
font: 12pt &quot;MS Shell Dlg 2&quot;;
}
QWidget{
background-color: rgb(66, 133, 200);
}</string>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <item>
                <widget class="QLabel" name="label_7">
                 <property name="minimumSize">
                  <size>
                   <width>160</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Cash + MPesa Sales : </string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="cashmpesaLbl">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>0.0</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="creditWdg" native="true">
              <property name="minimumSize">
               <size>
                <width>260</width>
                <height>40</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">*{
color: rgb(255, 255, 255);
font: 12pt &quot;MS Shell Dlg 2&quot;;
}
QWidget{
background-color: rgb(66, 133, 200);
}</string>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_6">
               <item>
                <widget class="QLabel" name="label_9">
                 <property name="minimumSize">
                  <size>
                   <width>160</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Credit Sales : </string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="creditLbl">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>0.0</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="chequeWdg_2" native="true">
              <property name="minimumSize">
               <size>
                <width>260</width>
                <height>40</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">*{
color: rgb(255, 255, 255);
font: 12pt &quot;MS Shell Dlg 2&quot;;
}
QWidget{
background-color: rgb(66, 133, 200);
}</string>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_11">
               <item>
                <widget class="QLabel" name="label_12">
                 <property name="minimumSize">
                  <size>
                   <width>160</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Credit Paid : </string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="creditpaidLbl">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>0.0</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QWidget" name="totalsWdg" native="true">
            <property name="minimumSize">
             <size>
              <width>260</width>
              <height>40</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">*{
font: 12pt &quot;MS Shell Dlg 2&quot;;
	color: rgb(222, 222, 222);
}
QWidget{
background-color: rgb(64, 129, 95);
}</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_8">
             <item>
              <widget class="QLabel" name="label_13">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>35</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string>Totals : </string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="totalsLbl">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>35</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string>0.0</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="totalsWdg_2" native="true">
            <property name="minimumSize">
             <size>
              <width>260</width>
              <height>40</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">*{
font: 12pt &quot;MS Shell Dlg 2&quot;;
	color: rgb(222, 222, 222);
}
QWidget{
background-color: rgb(64, 129, 95);
}</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_14">
             <item>
              <widget class="QLabel" name="label_14">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>35</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string>Total Money Paid : </string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="totalspaidLbl">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>35</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string>0.0</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_10"/>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidget_sales">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::NoSelection</enum>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>150</number>
     </attribute>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>40</number>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderMinimumSectionSize">
      <number>30</number>
     </attribute>
     <column>
      <property name="text">
       <string>Product Name</string>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="foreground">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Selling Price</string>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="foreground">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Quantity</string>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="foreground">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Sub-Total</string>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="foreground">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QTreeWidget" name="treeWidget">
     <property name="minimumSize">
      <size>
       <width>700</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>2</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::NoSelection</enum>
     </property>
     <attribute name="headerDefaultSectionSize">
      <number>140</number>
     </attribute>
     <column>
      <property name="text">
       <string>Product Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Selling Price</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Quantity</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Sub-Total</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
